pkg/formatter/stylish.pipeline.go

```diff
@@ -0,0 +1,52 @@
+package formatter
```

Comment:
Same thing with the dot (for stylish.k8s too) - it indicates to me the need of a different package 🤷‍♂️ .

So maybe move these to `pkg/pipeline` in `stylish.go` - or have a `pkg/stylish` package containing `pipeline.go`, `k8s.go`, or nest the formatter package into `formatter/stylish/[model].go`

Reply:
I like the idea of `formatter/stylish/[model].go`, at least for now feels like the simplest solution to break the stylish file a bit